config/common-packages.yml

```diff
@@ -15,22 +15,20 @@ install:
   - mediainfo
   # for auditing
   - lynis
-  - gnome-disk-utility
+  - chkrootkit
+  # other packages
   - headsetcontrol
   - mpv
-  - protonvpn-cli
   - qbittorrent
   - gimp
   - hardened_malloc
-  - chkrootkit
   - koji
   - codium
   
 remove:
   - firefox
   - firefox-langpacks
   - virtualbox-guest-additions
-  - openssh-server
```

Comment:
please revert this as well. the openssh server package should not be included. the images include the openssh client.

Reply:
why would this be a problem? The server will not be active by default and the firewall will block it